server/migrations/20211025130133-add-notional.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction(async (t) => {
+      // Add Notional
+      await queryInterface.bulkInsert(
+        'Chains',
+        [
+          {
+            id: 'notional',
+            symbol: 'NOTE',
+            name: 'Notional',
+            type: 'dao',
+            network: 'compound',
+            base: 'ethereum',
+            snapshot: 'notional.eth',
+            active: true,
+          },
+        ],
+        { transaction: t }
+      );
+
+      await queryInterface.bulkInsert(
+        'ChainNodes',
+        [
+          {
+            chain: 'notional',
+            url: 'wss://mainnet.infura.io/ws',
+            address: '0x086b4ecD75c494dD36641195E89c25373E06d7cB', // GovernorAlpha
+            token_name: 'comp', // for alpha/bravo
```

Comment:
should be `note` instead of `comp` based on https://etherscan.io/address/0x086b4ecD75c494dD36641195E89c25373E06d7cB#code